src/exports.h

```diff
@@ -11,6 +11,10 @@
 #define TJFLAG_FASTDCT 0
 #endif
 
+#ifdef DEFAULT_QUALITY
+#undef DEFAULT_QUALITY
```

Comment:
I already thought of this, but since I was already changing quite a bit I didn't want to touch something that I could get around